oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticBulkProcessorHandler.java

```diff
@@ -96,11 +96,13 @@ class ElasticBulkProcessorHandler {
     private ElasticBulkProcessorHandler(@NotNull ElasticConnection elasticConnection,
                                         @NotNull String indexName,
                                         @NotNull ElasticIndexDefinition indexDefinition,
-                                        @NotNull NodeBuilder definitionBuilder) {
+                                        @NotNull NodeBuilder definitionBuilder,
+                                        @NotNull boolean waitForESAcknowledgement) {
```

Comment:
primitive should not be annotated with `@NotNull`: remove the annotationor use Boolean